src/it/resources/application.conf

```diff
@@ -0,0 +1,8 @@
+process {
+  id = "test"
+}
+
+akka {
+  log-dead-letters = 0
+  loglevel = "ERROR"
```

Comment:
Ah sorry, I didn't realize this was in the "it" resources.

Also, did you mean "eigenflow uses warnings to report retries"?  That would seem more appropriate.